### PR TITLE
Updated breaking changes

### LINKED ISSUE #20327: Breaking change to extensions API endpoint
### Describe the Bug

In Directus version 10.6, it was possible to `GET /extensions/:type` to retrieve a list of enabled extensions. No permissions were needed except a valid static API token. In our app, we used this to validate the configuration of our instance. For example, we wanted to ensure that a custom hook extension was installed before allowing the request from a related feature.

However, in Directus version 10.7, this endpoint no longer exists, although there is the `GET /extensions` endpoint that returns all extensions along with their types and statuses. This is a nice improvement, except for one thing: [the `admin` flag is required on the account](https://github.com/directus/directus/blob/v10.7.2/api/src/services/extensions.ts#L50). I understand the need to secure this endpoint, but requiring the `admin` flag effectively breaks this endpoint for third party consumers. It would be more secure to require a read permission on the `directus_extensions` collection instead so that a third party API consumer could still use this endpoint with a limited set of permissions.

### To Reproduce

Run `curl -H "Authorization: Bearer {{token}}" http://localhost:8055/extensions` where `{{token}}` is a static token for user with a "service role" e.g. for a frontend app. Directus returns a 403, even if the role has permission to read `directus_extensions`. Granting the required `admin` status to a service role is much too broad, and renders the endpoint useless for third parties.

### Directus Version

v10.7.2

### Hosting Strategy

Directus Cloud

### PR DESCRIPTION
**Pull request by phazonoverload**
Fixes missing breaking change in #20327 and clarifies docs written in #20499

Closes #20327

### CODE REVIEW

File: docs/releases/breaking-changes.md [hunk 1]
```diff
@@ -15,13 +15,15 @@ Starting with Directus 10.0, here is a list of potential breaking changes with r
 
 ## Version 10.8.3
 
-### Updated GraphQL content version usage
+### Updated GraphQL Content Version Usage
 
-Content versioning was originally implemented as a GraphQL parameter however the output format for relational fields is
-different for stored versions resulting in a GraphQL error as this is unsupported.
+Previously when accessing content versions via GraphQL, a `version` parameter on existing fields. This has now been
```
Comment (paescuj) [inline docs/releases/breaking-changes.md:20]: This sentence seems incomplete

Comment (phazonoverload) [reply docs/releases/breaking-changes.md:20]: It's finished on the next line, as per the linter

Comment (paescuj) [reply docs/releases/breaking-changes.md:20]: > Previously when accessing content versions via GraphQL, a `version` parameter on existing fields.

Hmm, I'm missing a predicate here, for example:

> Previously when accessing content versions via GraphQL, a `version` parameter on existing fields **was used**.

Comment (phazonoverload) [reply docs/releases/breaking-changes.md:20]: Oh. Um. Nothing to see here, just my lack of ability to read.

Comment (paescuj) [reply docs/releases/breaking-changes.md]: Admittedly, GitHub diffs are not exactly the best tool for reviewing prose text 😅

**Review by paescuj — APPROVED** [review]
❤️

**Pull request merged**